requirements/base.txt

```diff
@@ -183,7 +181,7 @@ markupsafe==2.1.1
     # via jinja2
 mozilla-django-oidc==2.0.0
     # via mozilla-django-oidc-db
-mozilla-django-oidc-db==0.10.0
+mozilla-django-oidc-db==0.12.0
```

Comment:
Have you checked if there are any breaking changes between 0.10.0 and 0.12.0 that require us to deal with?